docs/python-examples/fixtures/conftest.py

```diff
@@ -0,0 +1,111 @@
+import logging
+
+import pytest
+
+import ocs
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def storage_class(request):
+    """
+    Storage class fixture
+
+    Returns:
+        StorageClass: object of storage class
+    """
+    sc_name = ocs.get_random_name('storage_class')
+    sc = ocs.StorageClass(sc_name)
+    logger.info(f"Creating storage class: {sc.name}")
+
+    def fin():
+        if not sc.is_deleted:
+            sc.delete()
+
+    request.addfinalizer(fin)
```

Comment:
This should be the first line being executed in the fixture

Reply:
done

Reply:
I thought as well but tested it and it really works. Just download the examples and run ;)